experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx

```diff
@@ -28,6 +28,9 @@ describe("SummaryTimeline", () => {
 
       expect(screen.queryByTestId("label-not-launched")).toBeInTheDocument();
       expect(screen.queryByTestId("label-start-date")).not.toBeInTheDocument();
+      expect(
+        screen.queryByTestId("label-release-date"),
+      ).not.toBeInTheDocument();
```

Comment:
`screen.queryByTestId(...)` returns `null` when not found so this really should be `expect(screen.queryByTestId(...)).toBeNull()`